taqueria-plugin-tezos-client/common.ts

```diff
@@ -0,0 +1,39 @@
+import { RequestArgs } from '@taqueria/node-sdk/types';
+import { join } from 'path';
+
+export interface TypeCheckOpts extends RequestArgs.ProxyRequestArgs {
+	sourceFile: string;
+}
+
+export interface SimulateOpts extends RequestArgs.ProxyRequestArgs {
+	sourceFile: string;
+	storage?: string;
+	param: string;
+	entrypoint?: string;
+}
+
+// To be used for the main entrypoint of the plugin
+export type IntersectionOpts = TypeCheckOpts & SimulateOpts;
+
+// To be used for common functions in this file
+type UnionOpts = TypeCheckOpts | SimulateOpts;
+
+export const FLEXTESA_IMAGE = 'oxheadalpha/flextesa:latest';
```

Comment:
I recall hearing that this change is temporary, but should we use the `latest` tag here? I think it might be best to fetch the latest version from dockerhub, and use that instead.